packages/airnode-utilities/hardhat.config.ts

```diff
@@ -0,0 +1,13 @@
+import { HardhatUserConfig } from 'hardhat/types';
```

Comment:
Hmm, this is necessary for the E2E tests, right? Can't we do it the same way as the `airnode-node` E2E tests? The blockchain preparation is implemented in `airnode-operation` and included. Can we do the same here?

Reply:
Related to using the airnode-validator types in airnode-utilities, it turns out that airnode-operations also uses airnode-utilities as a reference so adding them to each other would also cause a circular dependency.

What do you think?

Reply:
Hmm, that sucks. Ok, keep it as it is, for now, I've created an issue for this https://github.com/api3dao/airnode/issues/1327. Once https://github.com/api3dao/airnode/issues/1312 is done we will be able to rework it.